core/base/explicitTriangulation/ExplicitTriangulation.h

```diff
@@ -1004,20 +1000,47 @@ namespace ttk {
       return 0;
     }
 
+#ifdef CELL_ARRAY_NEW
```

Comment:
can you point me to where this variable is defined?
shouldn't we give it a more explicit name (like TTK_CELL_ARRAY_NEW)?

Reply:
ok, I found where it was defined (core/base/skeleton/CMakeLists.txt).
however, the explicit name would be better (thanks for pushing that!)